src/main/java/sirius/biz/jobs/batch/file/XMLExportJobFactory.java

```diff
@@ -20,16 +20,15 @@
  */
 public abstract class XMLExportJobFactory extends FileExportJobFactory {
 
-    protected final Parameter<Boolean> requireValidFile =
```

Comment:
why not keep this parameter this is a simple boolean and commonly needs no customization..

Reply:
parameter still exist, just the scope changes. It's created in L28 if xsd paths were provided.
https://github.com/scireum/sirius-biz/blob/d66daa6a906c18b1ce3ef23c0d65e178233ebe6b/src/main/java/sirius/biz/jobs/batch/file/XMLExportJobFactory.java#L28